md/forms.py

```diff
@@ -1,14 +1,119 @@
+import datetime
+
 from django import forms
-from selectable.forms import AutoCompleteSelectField
 
+from django.db.models import Q
+from selectable.forms import AutoCompleteWidget, AutoCompleteSelectField
+
+from md import models as stops
```

Comment:
I see that the NC code does that, but that was probably just a kludge to keep things working when an app was renamed.  Just import the symbols needed (from md.models import x, y, z)